pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -12,23 +14,23 @@ A product must exist in {ProductName} before you can assign a product name to th
 .Procedure
 
 . Navigate to menu:Products[New Product].
-
-. Enter the following information:
-  Product Name:: The text that appears in {ProductShortName}, the document preview, and in user-facing content.
-  Product UrlFragment:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
-  Product Description (optional):: A description of the product.
-
+. Enter the following information, items marked with a star are mandatory:
+  Product Name *:: The text that appears in {ProductShortName}, the document preview, and in user-facing content.
```

Comment:
I think this would be a good place to include a blurb like the following: "The product name is also injected into any associated document built by Pantheon via the :pantheonproduct: variable that is free for the author to use if they so choose."

That's probably pretty wordy but hopefully it gets the idea across; I'm sure you could come up with a better way to phrase it.